package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "axios-progress-bar",
-  "version": "1.2.0",
+  "version": "1.0.0",
```

Comment:
on line `10` repository should read `TechnologyAdvice/progress-bar-4-axios`

Reply:
think we need to change the name to something liking `technologyadvice/axios-progress-bar`, don't think it can hold the same namespace as the already existing repo